### ISSUE TITLE
Buttons are not screen reader friendly when using a keyboard

### ISSUE THREAD

**Issue by Malvoz** [GISdevio/OSM-Accessibility-Web#20]
I'm not sure what this project is for, but I found it through a reference in a Leaflet issue. 😋

I just want to note that the `<button>` controls in https://wikimedia-osm-disability.web.app/ are missing [_accessible names_](https://www.w3.org/TR/accname-1.1/#dfn-accessible-name), this is not very problematic for users with a mouse pointer because they can see the icons _and_ the tooltips let screen reader users know what they're used for. But the tooltips are not displayed when focusing the controls using a keyboard.

Here's the report on the accessibility issue: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwikimedia-osm-disability.web.app%2F#accessibility (generated from https://web.dev/measure).

It would probably be best to make sure that all controls are named properly (for example using [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)) even if the tooltips are displayed on hover/focus, but in that case screen readers would announce them twice, unless `role="tooltip"` was removed from the tooltip container.

**Comment by GISdevio-dev** [comment]
Dear @Malvoz,
thanks for the interest (though indirect) in this project and for reporting the accessibility issue.

We valued your feedback and proceeded to add the aria labels to the buttons with icons and to enable tooltips also to the keyboard focus and not only to the mouse hover.

**Comment by Malvoz** [comment]
Thanks for the fix, and sorry for the late response.

I'm re-using this issue to notify you that I wrote accessibility-related content for Leaflet, which you may find helpful:
- [Leaflet Guidelines to basic Leaflet accessibility](https://leafletjs.com/SlavaUkraini/examples/accessibility/)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)

**Comment by Malvoz** [comment]
Thanks for the fix, and sorry for the late response.

I'm re-using this issue simply to notify you that I wrote accessibility-related content for Leaflet, which you may find helpful:

- [Guidelines to basic Leaflet accessibility](https://leafletjs.com/SlavaUkraini/examples/accessibility/)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
